### ISSUE TITLE
Classif loss functions: Check which exponential is correct

### ISSUE THREAD

**Issue by jemus42** [PlantedML/randomPlantedForest#31]
Check back with @JosephTMeyer about exponential loss.

Affects lines
- https://github.com/PlantedML/randomPlantedForest/blob/5a4bcad7bb297d4ac8a76da09af57a0f439591aa/R/rpf.R#L110
- And test here https://github.com/PlantedML/randomPlantedForest/blob/5a4bcad7bb297d4ac8a76da09af57a0f439591aa/tests/testthat/test-predict-multiclass.R#L119

Other cases where `loss` is checked expect it to be in the set `c("logit", "logit_2", "exponential", "exponential_2")`, so there's no immediate need to remove unneeded loss functions there.

**Comment by jemus42** [comment]
Joseph confirmed `exponential` for multiclass is fine, meaning the current state is ok.
